### ISSUE TITLE
--ignore flag not working

### ISSUE THREAD

**Issue by maiers** [kmagiera/babel-watch#106]
It looks like `babel` (v7.5.4) has no `util` object

### To reproduce: 

call `babel-watch --ignore ./node_modules <file>`

### Actual outcome:

```
./node_modules/babel-watch/babel-watch.js:377
    return babel.util.shouldIgnore(filename, ignore || [], only);
```

### Expected outcome:

Requested expression gets ignored by babel

**Comment by jonas-db** [comment]
I have this problem! Workaround please @STRML ?

**Comment by adammichaelwilliams** [comment]
This is due to a [change in Babel 7](https://github.com/babel/babel/pull/5487).

**Comment by STRML** [comment]
Fixed in `7.2.0`.
